### PR TITLE
Epic/avo 1740

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/projects/AVO/versions/10052/tab/release-report-all-issues

https://github.com/viaacode/avo2-proxy/pull/432

### CODE REVIEW

**Comment by bertyhell** [comment]
deleted lock file?

File: src/admin/shared/components/FilterTable/FilterTable.utils.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { isArray, isEmpty, isNil, isPlainObject, isString, omitBy } from 'lodash';
```
Comment (bertyhell) [inline src/admin/shared/components/FilterTable/FilterTable.utils.ts:1]: prefer lodash-es, it's better tree shakeable
https://til.hashrocket.com/posts/dnzttruljf-prefer-lodash-es-when-using-webpack

File: src/admin/shared/components/FilterTable/FilterTable.const.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { compact, fromPairs } from 'lodash';
```
[the remark this answers is not in the document]
Comment (ian-emsens-shd) [reply src/admin/shared/components/FilterTable/FilterTable.const.ts:1]: https://meemoo.atlassian.net/browse/AVO-1947

File: src/admin/shared/components/FilterTable/FilterTable.utils.ts (hunk 1 above)
Comment (ian-emsens-shd) [reply src/admin/shared/components/FilterTable/FilterTable.utils.ts:1]: https://meemoo.atlassian.net/browse/AVO-1947

File: src/workspace/views/QuickLaneOverview.tsx [hunk 3]
```diff
@@ -1,14 +1,16 @@
 import { isEqual } from 'lodash';
```
[the remark this answers is not in the document]
Comment (ian-emsens-shd) [reply src/workspace/views/QuickLaneOverview.tsx:1]: https://meemoo.atlassian.net/browse/AVO-1947

**Comment by ian-emsens-shd** [comment]
> deleted lock file?

`
100 additions, 26,395 deletions not shown because the diff is too large. Please use a local Git client to view these changes.
`

**Pull request merged**